### DISCUSSION TITLE
Theme Wallbash Delay

### DISCUSSION THREAD

**Discussion by LucaMVieira** [prasanthrangan/hyprdots#1028] [Q&A] [answered]
My system broke a while back and I really enjoyed using hyprdots. I rebuild my system with Hyprdots again but this time whenever I change the theme, wallbash takes at least 2 min to go to the new theme. Also whenever I go to a new theme the wallpapers take the same time as the wallbash to load.

I have tried using "$HOME/Hyprdots/Scripts/create_cache.sh" to at least make all the wallpapers load when the pc start but is just says "zsh: command_not found"

Any thing I might do to make the wallbash and wallpapers to load instantly after changing themes?

![240306_18h09m48s_screenshot](https://github.com/prasanthrangan/hyprdots/assets/154270406/87e78369-47eb-40ee-8af4-14b4ba1539eb)
 
As  you can see the wallbash is not the same as the theme (wallbash still in one-dark, wallpaper Revan) and the wallpapers don't load

**Comment by kRHYME7** [comment]
Try to rebuild your Cache ``` Hyprdots cache rebuild ``` it will take time. Also that script you mentioned is not for executing at start.

**Reply by LucaMVieira** [reply]
I rebuild my cache and it did not work 

In this screenshot you can see better that the theme i selected was rose-pine but the them selector shows I`m still on Catppuccin-Latte  

![240306_20h30m57s_screenshot](https://github.com/prasanthrangan/hyprdots/assets/154270406/df2a1518-974e-4195-a1b0-1ac9b9b9179f)

**Reply by kRHYME7** [reply]
Try to toggle your theme next just to refresh all configurations.  (You can use your keybind ) 
or
``` Hyprdots theme next ```

Maybe do it multiple times.


Also can you provide the output for that too.

**Comment by cheeckhicks** [comment, marked as answer]
I just solved the problem for me. In my case I had no themes and wallpapers took like 5 sec to change. I cloned the repo, opened hyprdots/Scripts than did:
`bash themepatcher.sh "One-Dark" "https://github.com/RAprogramm/hyprdots-theme/tree/One-Dark"`
`bash themepatcher.sh "Synth-Wave" "https://github.com/prasanthrangan/hyprdots-mod/tree/Synth-Wave"`
`bash themepatcher.sh "Nordic-Blue" "https://github.com/prasanthrangan/hyprdots-mod/tree/Nordic-Blue""`
I think themes weren't installed properly or wasn't cached. Idk I'm a total beginner but this worked for me.
